Tests/LibGfx/TestImageDecoder.cpp

```diff
@@ -688,6 +688,19 @@ TEST_CASE(test_tiff_16_bits)
     EXPECT_EQ(frame.image->get_pixel(60, 75), Gfx::Color::NamedColor::Red);
 }
 
+TEST_CASE(test_tiff_cmyk)
+{
+    auto file = TRY_OR_FAIL(Core::MappedFile::map(TEST_INPUT("tiff/cmyk.tiff"sv)));
```

Comment:
I suppose this was just created in gimp? (maybe say in commit message.) Being able to save CMYK jpegs is "needs prerelease version" new; has it been around longer for tiff?

Reply:
> Being able to save CMYK jpegs is "needs prerelease version" new

Oh nice, I didn't know that!
https://github.com/SerenityOS/serenity/pull/22844 might or might not be related :smile: